concepts/fluid-balance/adjusted-urine-output/AllUrineOutputs.sql

```diff
@@ -0,0 +1,46 @@
+-- First we drop the table if it exists
+CREATE OR REPLACE TABLE `mimic-iii-256413.Tables.AllUrineOutputs` as
```

Comment:
The general style I've used is snake case for table names and kebab case for file names.